services/QuillLMS/config/routes.rb

```diff
@@ -475,7 +476,8 @@
   get '/sign-up/add-non-k12', to: 'accounts#new'
 
   namespace :auth do
-    get "/google_oauth2/callback" => 'google#google'
+    get '/google_oauth2/callback' => 'google#authorized_and_authenticated'
```

Comment:
Note: omniauth 2.x, which we want to upgrade to soon, only accepts POSTs for oauth callbacks: https://github.com/omniauth/omniauth/commit/b72a8db667960ae1a930d0df4f5e5917dcd0ad7a